src/core/dev_api/openvino/core/meta_data.hpp

```diff
@@ -32,4 +33,14 @@ class OPENVINO_API Meta {
     virtual ~Meta() = default;
 };
 
+class MetaDataWithPugixml : public Meta {
+public:
+    /**
+     * @brief Returns meta unchanged meta information. Throws ov::Exception if the meta was potentially changed
+     *
+     * @return const pugi::xml_node& with meta information
+     */
+    virtual pugi::xml_node get_pugi_node() const = 0;
```

Comment:
do you think we can return const reference to avoid including of pugixml.hpp? We could use forward declaration for pugi classes.

Reply:
done